### PR TITLE
und-746 mobile menu changes

### PR DESCRIPTION
**Pull request by Ashish-srij**
mobile menu changes

### CODE REVIEW

File: stories/assets/js/navigation.js [hunk 1]
```diff
@@ -42,7 +42,9 @@ export const init = (locale) => {
   $(document).on('click', '.mob-links .cta__link', function () {
     const id = $(this).attr('id');
     const text = $(this).text();
+    console.log(text)
```
Comment (shishirphadke) [inline stories/assets/js/navigation.js:45]: @Ashish-srij Remove `console.log()` here.

File: stories/assets/js/navigationData.js [hunk 2]
```diff
@@ -5,11 +5,13 @@ export const getData = (lang) => {
     url: './js/navigationData.json',
     dataType: 'json',
     success(res) {
+      lang=lang?lang:'english'
```
Comment (shishirphadke) [inline stories/assets/js/navigationData.js:8]: @Ashish-srij Add semicolon at the end here. Also format the code accordingly.

File: stories/assets/js/navigationData.js [hunk 3]
```diff
@@ -5,11 +5,13 @@ export const getData = (lang) => {
...
       const resData = res.find((item) => item.language === lang);
       const megaWrapper = $('.mega-wrapper');
       const mobileMegaWrapper = $('.mobile-mega-wrapper');
       megaWrapper.empty();
       mobileMegaWrapper.empty();
+      console.log('Data ',resData);
```
Comment (shishirphadke) [inline stories/assets/js/navigationData.js:14]: @Ashish-srij Remove `console.log()` here.

File: stories/assets/js/navigation.js [hunk 4]
```diff
@@ -42,7 +42,9 @@ export const init = (locale) => {
...
     $('.mobile-mega-content').find('.sub-heading').text(text);
+    console.log($('.mobile-mega-wrapper').find(`[data-mobile-id='${id}']`));
```
Comment (shishirphadke) [inline stories/assets/js/navigation.js:47]: @Ashish-srij Remove `console.log()` here.

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**